dhash/module.py

```diff
@@ -18,20 +21,41 @@
 LIMIT_SOFT = 14
 
 MAX_ATTACHMENT_SIZE = 8000
+ALLOWED_FORMATS = ("jpg", "jpeg", "png", "webp", "gif")
+
+HTTP_HEADERS = {
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36"
+}
+
+URL_REGEX = r"(https?://[^\s]+)"
 
 
 class Dhash(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
         self.embed_cache = {}
 
+        HashConfig.init(
+            "allowed_urls",
+            "^https://(?:cdn\\.discordapp\\.com|media\\.discordapp\\.net)/",
+        )
```

Comment:
It is enabled by default. And if administrator decides to change it, he can. I would personnaly turn it off instantly. Also, I can image scenarios, where you would want to scan images from your own domain, from Reddit or whatever and in case it's hardcoded, you would have to change that code and get into trouble with updates and so on.

Reply:
After Discord discussion I've finally understood.